### ISSUE TITLE
Invalid serialized string

### ISSUE THREAD

**Issue by gedaiu** [rejectedsoftware/vibe.d#1620]
This test outputs an invalid json string:
```
unittest {
	struct Point
	{
		float[2] coordinates;
	}

	Point().serializeToJson.toPrettyString.writeln;
}
```

```
{
	"coordinates": [
		undefined,
		undefined
	]
}
```

**Comment by gedaiu** [comment]
I think there should be `null` instead of `undefined`

**Comment by s-ludwig** [comment]
For which version does this occur? This is supposed to be fixed since #1548 was merged (contained in 0.7.30).

**Comment by gedaiu** [comment]
`vibe-d:data 0.7.30-alpha.3+commit.30.g82b9d0e` if it's fixed, I think we can close this
